part-2-server/server.js

```diff
@@ -0,0 +1,38 @@
+var http = require ("http");
+var fs = require("fs");
+
+var server = http.createServer(function (req, res) {
+	var url = req.url;
+	if (url === "/") {
+		fs.readFile("index.html", function (err, data) {
+			if (err) throw err;
+			res.write(data.toString());
+			res.end();
+		});
+	} else if (url === "/page-2" || url === "/page-3") {
+		fs.readFile("page-2.html", function (err, data) {
+			if (err) throw err;
+			res.write(data.toString());
+			res.end();
+		});
+	} else {
+		// res.write("<h1>404</h1>");
+		// res.write("<h3>Page Not Found</h3>");
+		// res.end();
+		for (var i = 0; i < 140; i++) {
```

Comment:
Don't need a loop here but this will actually work, at least for getting the url. Could have also just split the url into two substrings, the first substring being 'poem-' and the second substring being the number. Could then use the first substring to match against the url, and second substring to look up the correct file